City-of-Helsinki/drupal-palvelukeskus
# UHF-8401: Matkapalvelu chat

# [UHF-8401](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8401)
Add Matkapalvelu chat

## What was done
Added an implementation of Matkapalvelu (Genesys) chat as a custom block.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-8401-matkapalvelu-chat`
  * `make fresh`
* Run `drush locale:check && drush locale:update`
* Run `make drush-cr`

## How to test

* Go to block layout
* Add "Palvelukeskus Genesys chat" block to a page of your choosing
* View the page. You should see the chat in the bottom right corner
* Open the chat and test if it works

## Designers review





[UHF-8401]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8401?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
